connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -106,6 +108,17 @@ public void stop() {
         Utils.closeQuietly(targetAdminClient, "target admin client");
     }
 
+    @Override
+    public Config validate(Map<String, String> connectorConfigs) {
+        List<ConfigValue> configValues = super.validate(connectorConfigs).configValues();
+        MirrorCheckpointConfig.validate(connectorConfigs).forEach((config, errorMsg) ->
+                configValues.stream()
+                        .filter(conf -> conf.name().equals(config))
```

Comment:
`EMIT_OFFSET_SYNCS_ENABLED` is in `MirrorSourceConfig`, so it is not a part of config def of `MirrorCheckpointConnector`. Hence, the error related to `EMIT_OFFSET_SYNCS_ENABLED` can't be propagated.

Reply:
The following test shows my concern:
```java
    @Test
    public void test() {
        Map<String, String> props = new HashMap<>();
        props.put(MirrorConnectorConfig.EMIT_OFFSET_SYNCS_ENABLED, "false");
        MirrorCheckpointConnector connector = new MirrorCheckpointConnector();
        Config config = connector.validate(props);
        assertEquals(1, config.configValues().stream().filter(c -> c.name().equals(MirrorConnectorConfig.EMIT_OFFSET_SYNCS_ENABLED)).count());
    }
```

Reply:
I moved this to MirrorConnectorConfig which all inherit. So this should be addressed now

Reply:
I still agree with @mimaison that we shouldn't have this property in the `MirrorConnectorConfig` class, since then it'll be included as a "common config" for all connectors in our generated docs [here](https://kafka.apache.org/37/documentation.html#mirrormakercommonconfigs) instead of in the `MirrorSourceConnector`-specific docs [here](https://kafka.apache.org/37/documentation.html#mirrormakersourceconfigs).

Can we just add a new `ConfigValue` for the property in `MirrorCheckpointConnector::validate` if we find a validation error with it?

Reply:
I updated `MirrorCheckpointConnector::validate` to create configValue if the name of the config not defined

Reply:
No worries! Latest looks good to me 👍

@chia7712 thoughts?

Reply:
LGTM